UKHomeOffice/sgmr-service
# Jest testing sgmr

## Description
Updates jest and adds testing setup for jest with react-testing-library. This includes adding npm packages 'regenerator-runtime' and 'msw' for asynchronous testing. Also, adds a test for the UserRegisterValidation component.

### To test
1. Pull the branch
2. Ensure `react-remove-properties` plugin is commented out in .babelrc (prevents removal of `data-testid` property in html, comment back in for removal in prod)
3. Run `npm run test`

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
